integrations/google-analytics/lib/index.js

```diff
@@ -580,6 +580,25 @@ function formatValue(value) {
   return Math.round(value);
 }
 
+function getProductPosition(item, products) {
+  var position = item.properties().position;
```

Comment:
This seems safe. But is there ever a time that `position` is a property that is not intended to be used in this way?

Reply:
Not by our spec. This position is an attribute of a a Product.